aws/resource_aws_lambda_event_source_mapping.go

```diff
@@ -191,6 +191,14 @@ func resourceAwsLambdaEventSourceMappingRead(d *schema.ResourceData, meta interf
 	d.Set("uuid", eventSourceMappingConfiguration.UUID)
 	d.Set("function_name", eventSourceMappingConfiguration.FunctionArn)
 
+	if *eventSourceMappingConfiguration.State == "Enabled" {
```

Comment:
Thanks for the pointer for `StringValue`! I had to be a bit more advanced for the `enabled` state though, otherwise it also maps `Creating`, `Updating`, `Deleting`  to false which breaks the tests (and is also not what we want).